Automattic/jetpack
# Jetpack Social: Disable Media Picker in sidebar when Social Image Generator is active

With this change we disable the media picker if SIG is enabled, to handle some confusion while we are not implementing the new UI.

More here: pdrWKz-TB-p2

## Proposed changes:

* Updated media section so it accepts a notice and a disabled prop
* Set the disabled prop based on SIG availability.

### Other information:


## Jetpack product discussion


pdrWKz-TB-p2
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Open the editor with the Advanced plan.
* You should no be able to pick/remove/interact with the media picker if SIG is enabled. Turning SIG off should re-enable the picker.

https://user-images.githubusercontent.com/36671565/234886786-35759da8-678e-4c01-a967-b1514609a0b3.mp4
